### DISCUSSION TITLE
How to display complete trajectory of the vehicle?

### DISCUSSION THREAD

**Discussion by wuhutatai** [eleurent/highway-env#421] [Q&A]
Thank for your wonderful work! Here are some questions:
First question is:
When I set config of parking-env "show_trajectories": True
the earlier trajectories disappeared when trajectory is too long ,so how to keep the trajectories from disappearing to show the 
complete trajectory of the vehicle
 
Another question is :
how to output the state of the vehicle for making pictures?

**Comment by eleurent** [comment]
Hi,
>so how to keep the trajectories from disappearing to show the complete trajectory of the vehicle

You can increase this value a lot:

https://github.com/eleurent/highway-env/blob/abcef8da79dedf0dfdbe2db5b176d6f203366489/highway_env/vehicle/kinematics.py#L31

>  how to output the state of the vehicle for making pictures?

Not sure what you need exactly, but you can access the controlled vehicle with `env.vehicle`, and then access its field like position, heading, speed etc.
